third-party/realdds/include/realdds/dds-option.h

```diff
@@ -208,4 +208,28 @@ class dds_ip_option : public dds_string_option
 };
 
 
+class dds_rect_option : public dds_option
+{
+    using super = dds_option;
+
+public:
+    struct type
+    {
+        int x1, y1;
+        int x2, y2;
+
+        rsutils::json to_json() const;
+        static type from_json( rsutils::json const & j ) { return { j[0], j[1], j[2], j[3] }; }
+    };
+
+public:
```

Comment:
Why the second `public`?

Reply:
No special reason, probably just a leftover - will remove

Reply:
Actually, I think I like it - it separates "data types" from "API".